great_expectations/data_context/data_context.py

```diff
@@ -2358,6 +2398,22 @@ def save_expectation_suite(
         self._evaluation_parameter_dependencies_compiled = False
         return self.expectations_store.set(key, expectation_suite, **kwargs)
 
+    def _save_expectation_suite(
```

Comment:
Can we use the public method instead? Or are we intending to deprecate it?

Reply:
Upon further review, I removed the code that was calling this regardless, so I will remove this method. I initially wrote it because I have an aversion to using public API methods internally even if it does the same thing. ✅